src/components/Table.js

```diff
@@ -234,7 +234,7 @@ export default class Table extends React.Component<TableDefaultProps, TableProps
       this.setState({
         activeRowIndex,
         anchorRowIndex: activeRowIndex,
-        selectedRowIndices: sortedRows.length > 0 ? new Set(selectedRows) : new Set(),
+        selectedIndices: noEmptySelection ? new Set([0]) : new Set([]),
```

Comment:
What's the deal with this line? Don't you want to only set selectedIndices to `new Set[0]` if there are actually rows in the table?

Reply:
If there are no rows in the table, then having `selectedIndices` set to `new Set([0])` should have no effect.
I think it's safe to do this. It's the way `selectedIndices` is initialized in the constructor on line 209 above.

Reply:
OK